src/pages/IstioRulesList/IstioRuleListComponent.tsx

```diff
@@ -147,7 +147,11 @@ class IstioRuleListComponent extends React.Component<IstioRuleListComponentProps
         })
         .catch(namespacesError => {
           console.error(JSON.stringify(namespacesError));
-          this.handleError('Error fetching namespace list.');
+          let errorMessage = 'Could not fetch namespace list.';
+          if (namespacesError['response']['data'] && namespacesError['response']['data']['error']) {
+            errorMessage = errorMessage + ' Error: [ ' + namespacesError['response']['data']['error'] + ' ]';
+          }
```

Comment:
This seems copy-pasted everywhere, can it be extracted somewhere? Something like `errorMessageFor('Could not fetch namespace list.')`.